### PR TITLE
Add unit test troubleshoot doc.

### LINKED ISSUE #1239: Local test run document update.
### Issue description

The guide to run test cases locally didn't work as expected in ubuntu environment.

Would like to update the unit test setup guide

* apisix version (cmd: `apisix version`):
* OS: Ubuntu 18.04

### PR DESCRIPTION
**Pull request by Akayeshmantha**
Modify Unit test local run setup document.

### Issues resolved

Fix #1239

### CODE REVIEW

**Comment by Akayeshmantha** [comment]
@membphis  @moonming

**Comment by membphis** [comment]
@Akayeshmantha please take a look at the Travis output: https://travis-ci.org/github/apache/incubator-apisix/jobs/661963450#L689

And I have created a new issue about the Chinese version: 

https://github.com/apache/incubator-apisix/issues/1255

**Comment by membphis** [comment]
Wow, you updated the Chinese documentation by yourself. I want other Chinese developers to do this for you, but you have done it all.

It shocked me, you are awesome.

**Comment by Akayeshmantha** [comment]
Hi @membphis  I think this issue now in travis is a one time error 

`+export PATH=/usr/local/openresty-debug/nginx/sbin:/usr/local/openresty-debug/luajit/bin:/usr/local/openresty-debug/bin:/home/travis/.rvm/gems/ruby-2.5.3/bin:/home/travis/.rvm/gems/ruby-2.5.3@global/bin:/home/travis/.rvm/rubies/ruby-2.5.3/bin:/home/travis/.rvm/bin:/home/travis/bin:/home/travis/.local/bin:/usr/local/lib/jvm/openjdk11/bin:/opt/pyenv/shims:/home/travis/.phpenv/shims:/home/travis/perl5/perlbrew/bin:/home/travis/.nvm/versions/node/v8.12.0/bin:/home/travis/gopath/bin:/home/travis/.gimme/versions/go1.11.1.linux.amd64/bin:/usr/local/maven-3.6.0/bin:/usr/local/cmake-3.12.4/bin:/usr/local/clang-7.0.0/bin:/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin:/snap/bin:/home/travis/.phpenv/bin:/opt/pyenv/bin:/home/travis/.yarn/bin
+PATH=/usr/local/openresty-debug/nginx/sbin:/usr/local/openresty-debug/luajit/bin:/usr/local/openresty-debug/bin:/home/travis/.rvm/gems/ruby-2.5.3/bin:/home/travis/.rvm/gems/ruby-2.5.3@global/bin:/home/travis/.rvm/rubies/ruby-2.5.3/bin:/home/travis/.rvm/bin:/home/travis/bin:/home/travis/.local/bin:/usr/local/lib/jvm/openjdk11/bin:/opt/pyenv/shims:/home/travis/.phpenv/shims:/home/travis/perl5/perlbrew/bin:/home/travis/.nvm/versions/node/v8.12.0/bin:/home/travis/gopath/bin:/home/travis/.gimme/versions/go1.11.1.linux.amd64/bin:/usr/local/maven-3.6.0/bin:/usr/local/cmake-3.12.4/bin:/usr/local/clang-7.0.0/bin:/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin:/snap/bin:/home/travis/.phpenv/bin:/opt/pyenv/bin:/home/travis/.yarn/bin
+openresty -V
openresty: error while loading shared libraries: libluajit-5.1.so.2: cannot open shared object file: No such file or directory
The command "$PWD/.travis/${OSNAME}_runner.sh script" exited with 127.`

Since I havent added any config changes it's impossible to have nan error like this wdyt ?

**Comment by membphis** [comment]
it should be a bug of Travis, I have restarted this job, you can wait for more time.

**Comment by Akayeshmantha** [comment]
@membphis  any thoughts ?

**Comment by membphis** [comment]
Failed again, I am checking.

File: doc/how-to-build.md [hunk 1]
```diff
@@ -106,12 +106,19 @@ Makefile rules:
 
 1. Install perl's package manager `cpanminus` first
 2. Then install `test-gninx` via `cpanm`:：`sudo cpanm --notest Test::Nginx IPC::Run > build.log 2>&1 || (cat build.log && exit 1)`
-3. Clone source code：`git clone https://github.com/openresty/test-nginx.git`;
+3. Clone source code：`git clone https://github.com/apache/incubator-apisix.git`;
```
Comment (membphis) [inline doc/how-to-build.md]: The old version is right. The `apisix` and `test-nginx` should be put in the same folder.

```
$ ls 
apisix       test-nginx
```

Comment (Akayeshmantha) [reply doc/how-to-build.md]: Hi @membphis  sure will change accordingly.

**Comment by membphis** [comment]
@Akayeshmantha many thanks, merged

**Pull request merged**